pkg/autodiscovery/configresolver.go

```diff
@@ -38,34 +37,32 @@ var (
 // services it hears about with templates to create valid configs.
 // It is also responsible to send scheduling orders to AutoConfig
 type ConfigResolver struct {
-	ac              *AutoConfig
-	collector       *collector.Collector
-	templates       *TemplateCache
-	services        map[listeners.ID]listeners.Service // Service.ID --> []Service
-	serviceToChecks map[listeners.ID][]check.ID        // Service.ID --> []CheckID
-	adIDToServices  map[string][]listeners.ID          // AD id --> services that have it
-	config2Service  map[string]listeners.ID            // config digest --> service ID
-	newService      chan listeners.Service
-	delService      chan listeners.Service
-	stop            chan bool
-	health          *health.Handle
-	m               sync.Mutex
+	ac             *AutoConfig
+	collector      *collector.Collector
+	templates      *TemplateCache
+	services       map[listeners.ID]listeners.Service // Service.ID --> []Service
+	adIDToServices map[string][]listeners.ID          // AD id --> services that have it
```

Comment:
Can we be consistent, use either `A2B` or `AToB` please ? I'd rather use `AToB` though.